frontend/simple/views/components/Lists/ListItem.vue

```diff
@@ -0,0 +1,146 @@
+<template>
+  <li class="c-item" :class="{ 'has-divider': hasDivider }">
+    <!--
+      QUESTION: The elements with .fa .c-item-text and .c-item-badge
+      are duplicated because I didn't found a clean way of dynamically change
+      their parents tag (router-link vs button vs a) and apply the custom attrs.
+
+      For now IMO "Duplication is better than the wrong abstraction"
+      until we found a solid solution for this problem
+    -->
```

Comment:
OK, for this can you create an issue and assign to @hubudibu?

Reply:
I've made this question at stackoverflow a few days ago and someone helped me! [Conditional Parent Element on Vue.js 2](https://stackoverflow.com/questions/51293489/conditional-parent-element-attrs-on-vue-js-2)